src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private int base;

```

Comment:
The variable names 'base' and 'side' are not very descriptive for a rectangle. Consider using 'width' and 'height' for clarity.